website/addons/s3/templates/s3_node_settings.mako

```diff
@@ -64,20 +73,6 @@
       </div>
       </div>
     </div>
-    <div data-bind="if: showCreateCredentials">
```

Comment:
Fixed this a different way. `showCreateCredentials` and `showTokenCreateButton` had duplicated logic, removed `showTokenCreateButton` and used `showCreateCredentials` where it should be used.